examples/e2etest.rs

```diff
@@ -0,0 +1,384 @@
+/*
```

Comment:
Examples should be separated from e2e tests, i suggest you use another directory as in other Astarte repos